### ISSUE TITLE
[UI] Deep link the expand scopes view

### ISSUE THREAD

**Issue by helfi92** [taskcluster/taskcluster#1598]
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

**Describe the bug**
It's hard to share links of the expand scopes view without having to re-type the scope.

**To Reproduce**
1. Go to https://taskcluster-ui.herokuapp.com/auth/scopes/expansions
2. Type the following in the editor

```
assume:repo:hg.mozilla.org/ci/ci-admin-try:*
assume:project-admin:ateam
```

3. Click the expand scopes button in the lower right
4. Refresh the page

**Actual behavior**
When the user refreshes the page, the editor becomes empty, and the expanded scopes that were displayed after clicking the expand scopes have all been cleared.

**Expected behavior**
When the user refreshes the page, the page looks like before:

<img width="1674" alt="Screen Shot 2019-10-13 at 10 31 51 AM" src="https://user-images.githubusercontent.com/3766511/66717248-ca8a6e00-eda4-11e9-9589-4c65067caaf6.png">

**Additional context**
We can store the editor value as a query parameter and use the [`qs`](https://www.npmjs.com/package/qs) package (already imported in the `ui` directory) to manipulate the query string. We can name the query parameter `scopes` and store it as an array.

**Comment by shilpiverma509** [comment]
@helfi92  Can I try working on this?

**Comment by naimiii** [comment]
@helfi92  can i contribute on it ?

**Comment by JiwoonKim** [comment]
Hi! I've completed all the issues assigned to me. May I take this issue?

**Comment by sccofield** [comment]
@helfi92 I am an outreachy intern, can I work on this?

**Comment by Rolikasi** [comment]
Hi, @helfi92 . It looks interesting, can I work on this issue?)

**Comment by shilpiverma509** [comment]
@helfi92  I have tried to fix this in #1696

**Comment by djmitche** [comment]
It looks like this is finished!
